spring-integration-core/src/main/java/org/springframework/integration/store/SimpleMessageStore.java

```diff
@@ -58,6 +58,12 @@
 	private volatile boolean isUsed;
 
 	private volatile boolean copyOnGet = false;
+	
+	private final int maxWaitInMilliseconds;
+	
+	private final static int DEFAULT_MAX_WAIT_IN_MILLISECONDS = 0;
+
+	private final static int DEFAULT_CAPACITY = 0;
```

Comment:
Well, since you are introducing `maxWait`, we won't have two of those 0s in the code below.
Having all of them in the ctors make it readable.
Plus good JavaDoc for the ctor with 0s make the sutf on the matter, too.

From other side it is just my position and I can live with these costants as well :smile: